proto/client-api/src/lib.rs

```diff
@@ -713,14 +716,17 @@ pub struct TrackPatchCommand {
 /// [`Event::PeerUpdated`].
 #[cfg_attr(feature = "client", derive(Deserialize))]
 #[cfg_attr(feature = "server", derive(Serialize))]
-#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub struct TrackPatchEvent {
     /// ID of the [`Track`] which should be patched.
     pub id: TrackId,
 
     /// General media exchange direction of the `Track`.
     pub media_direction: Option<MediaDirection>,
 
+    /// IDs of the `Member`s who should receive this outgoing [`Track`].
+    pub receivers: Option<Vec<MemberId>>,
```

Comment:
@logist322 why `Option<Vec<_>>` and not just `Vec`? Does `None` have a special meaning here? If so, it should be documented clearly.

Reply:
@tyranron yes, it does have. This field discribes **not changes**, but **an actual state**. `Some` means there are some changes in the `receivers` field (or we just want to sync `receivers` on the client side) and `None` means that we do not want to check and recalculate `receivers` on the client side.
It could be just `Vec<_>`, but in this case we must check and recalculate `receivers` on the client side on every gathered `TrackPatchEvent `.
Or we have to contain some struct in this field to describe changes (which `MemberId`s are added, which `MemberId` are deleted), but I do not think it is really needed.